client/src/components/AuthNavigation.js

```diff
@@ -91,13 +91,18 @@ const AuthNavigation = () => {
         return history.push("/profile");
     }
 
+    const goHome = () => {
+        setMatchState(null);
```

Comment:
Maybe make use of this reducer: https://github.com/hatchways/team-iron-man/pull/79/files#diff-06f4d0361f561dbd2650629c022fa3149776b79f8c68d6101eee7aa4c7c71bd9R74

Reply:
Oh that is for the user context, the fix here is for the match context. I don't have stuff like that set up for the match context right now